Makefile

```diff
@@ -61,7 +61,11 @@ aiohttp/%.c: aiohttp/%.pyx $(call to-hash,$(CYS)) aiohttp/_find_header.c
 .PHONY: cythonize
 cythonize: .install-cython $(PYXS:.pyx=.c)
 
-.install-deps: .install-cython $(PYXS:.pyx=.c) $(call to-hash,$(CYS) $(REQS))
+.compile-deps:
+	pip install pip-tools
+	pip-compile -q requirements/dev.in
```

Comment:
This shouldn't be done on each install. Each update should be performed in separate tested PRs.

Reply:
Done.